src/Data/Rational.purs

```diff
@@ -1,64 +1,17 @@
 module Data.Rational
   ( Rational
-  , runRational
-  , rational
-  , (%)
```

Comment:
Maybe it would be nice to re-export `%` so that people who have only imported Data.Rational don't need to adjust their imports as much?